vcs/VCS_Principles/VCS_Principles.ipynb

```diff
@@ -331,10 +285,10 @@
     "\n",
     "[Back to Top](#top)\n",
     "\n",
-    "Text object allow you insert text anywhere on the plot\n",
-    "Text object are made by combining two different secondary object: text orientation objects and text table objects\n",
+    "Text objects allow you insert text anywhere on the plot.\n",
```

Comment:
Change sentence to: Text objects allow you to insert text anywhere on the plot.
(It was missing the 'to').

Reply:
I just made the change - thanks for catching it!